packages/ws-server/src/webSocketServer.ts

```diff
@@ -120,6 +131,8 @@ app.ws.use((ctx) => {
     ctx.websocket.on('close', function () {
         relay.subs()?.unsubscribe(ctx.websocket);
         console.log('stopping client interval');
+        connectedClients--;
```

Comment:
Q: didn't we explore this earlier and find out that `connectedClients = ctx.app.server._connections;` captures increments and decrements?

If this is not the case we should add a comment saying decrements aren't handled.

Also should the decrement if applicable occur after the termination to avoid inconsistencies if termination doesn't occur.

Reply:
Since I place my test at the bottom it was failing due to the bug fixed on [PR](https://github.com/hashgraph/hedera-json-rpc-relay/pull/1017/files), but I have changed the order of the tests so I dont need to include that fix in my PR and tests dont fail either.